test/framework/filetools.py

```diff
@@ -1470,6 +1470,24 @@ def test_copy_dir(self):
         ft.mkdir(testdir)
         self.assertErrorRegex(EasyBuildError, "Target location .* already exists", ft.copy_dir, to_copy, testdir)
 
+        # if the directory already exists and 'dirs_exist_ok' is True, copy_dir should succeed
+        ft.copy_dir(to_copy, testdir, dirs_exist_ok=True)
+        self.assertTrue(sorted(os.listdir(to_copy)) == sorted(os.listdir(testdir)))
+
+        # if the directory already exists and 'dirs_exist_ok' is True and there is another named argument (ignore)
+        # we expect clean error on Python < 3.8 and pass the test on Python >= 3.8
+        # NOTE: reused ignore from previous test
+        shutil.rmtree(testdir)
+        ft.mkdir(testdir)
+        if (sys.version_info[0] == 3 and sys.version_info[1] >= 8) or sys.version_info[0] > 3:
+            ft.copy_dir(to_copy, testdir, dirs_exist_ok=True,
+                        ignore=lambda src, names: [x for x in names if '6.4.0-2.28' in x])
```

Comment:
this is more readable if you add a variable for the lambda function

Reply:
Then I need to define function otherwise it will not be according to PEP8 .